querystring/index.js

```diff
@@ -5,7 +5,7 @@ function stringify (obj) {
     if (isArray(val)) {
       var l = val.length
       for (var i = 0; i < l; i++) { result += '&' + key + '=' + val[i] }
-    } else {
+    } else if (val) {
```

Comment:
@abdulhannanali thanks so much for this pr, looks great. The only problem I can see here is that this will remove all falsey values, including zero and false, so here it would be better to do something like so:

``` if (val === void 0) { ```